tests/src/test/scala/common/Wsk.scala

```diff
@@ -408,9 +408,7 @@ class WskRule()
         val params = Seq(noun, if (!update) "create" else "update", "--auth", wp.authKey, fqn(name), (trigger), (action)) ++
             { annotations flatMap { p => Seq("-a", p._1, p._2.compactPrint) } } ++
             { shared map { s => Seq("--shared", if (s) "yes" else "no") } getOrElse Seq() }
-        val result = cli(wp.overrides ++ params, expectedExitCode)
-        if (expectedExitCode == SUCCESS_EXIT) assert(result.stdout.contains("ok:"), result)
```

Comment:
I just modified the rules class here to make it match the other classes in `Wsk.scala`. Meaning the other classes (action, trigger, etc) do not make assertions. Instead we rely on the actual tests to do so. And yes, the messages may change over time.